docs/community/highlights.md

```diff
@@ -6,6 +6,12 @@ keywords:
   - highlights
 ---
 
-The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions whether it's an article you wrote for [Codecademy's Docs website](https://www.codecademy.com/pages/contribute-docs), revising one of the lessons on [freeCodeCamp's curricula](https://contribute.freecodecamp.org/#/index?id=learning-platform), or creating a tutorial for people who use [Audacity](https://support.audacityteam.org/community/contributing/tutorials).
+The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions, share the story, and inspire others to join you in your open source journey. For maintainers, it's a great way to showcase your project, the issues that need support, and attract new contributors.
```

Comment:
[nitpick] I think some of the project links should be kept in this explanation. It could help people, especially contributors have some sort of idea on how they can start their journey. 
```suggestion
The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions, share the story, and inspire others to join you in your open source journey. For maintainers, it's a great way to showcase your project, the issues that need support, and attract new contributors.
```

Reply:
@CBID2 I didn't keep them because they don't reflect the implementation of the highlight.

If we want to include examples, it should be the PR, Issue, or Dev post.